### PR TITLE
Changes to keep align with the deployment guide

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: kubernetes/akka-cluster.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
         - name: appka
-          image: bootstrap-demo-kubernetes-api:1.3.3.7
+          image: akka-sample-cluster-kubernetes:latest
```
Comment (chbatey) [inline kubernetes/akka-cluster.yml:21]: fix name from when it was inside akka management

File: pom.xml [hunk 2]
```diff
@@ -22,37 +22,43 @@
     <akka-management.version>1.0.7</akka-management.version>
     <akka-http.version>10.1.12</akka-http.version>
     <scala.compat.version>2.13</scala.compat.version>
+    <version.number>${git.commit.time}.${git.commit.id.abbrev}</version.number>
```
Comment (chbatey) [inline pom.xml:25]: remove silly 1337 version

File: pom.xml [hunk 3]
```diff
@@ -76,101 +82,63 @@
       <version>${akka-management.version}</version>
     </dependency>
   </dependencies>
-
   <build>
     <plugins>
-      <plugin>
```
Comment (chbatey) [inline pom.xml:82]: no need to shade when we're just putting inside a Docker image

File: pom.xml [hunk 4]
```diff
@@ -76,101 +82,63 @@
...
+            <id>build-docker-image</id>
+            <phase>package</phase>
+            <goals>
+                <goal>build</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+          <groupId>pl.project13.maven</groupId>
```
Comment (chbatey) [inline pom.xml:126]: git based versioning

File: pom.xml [hunk 5]
```diff
@@ -22,37 +22,43 @@
     <akka-management.version>1.0.7</akka-management.version>
     <akka-http.version>10.1.12</akka-http.version>
     <scala.compat.version>2.13</scala.compat.version>
```
Comment (patriknw) [inline pom.xml:24]: do we use that property name `scala.compat.version` in many places? somewhat strange name in my opinion.

File: pom.xml (hunk 4 above)
Comment (ktoso) [reply pom.xml:126]: Oh hey there 👋 Nice to see the plugin here 😄

File: pom.xml (hunk 5 above)
Comment (chbatey) [reply pom.xml:24]: i'll switch that to just scala version

Comment (raboof) [reply pom.xml:24]: I think we usually use `scala.binary.version` nowadays

File: pom.xml (hunk 4 above)
Comment (chbatey) [reply pom.xml:126]: 👍 thanks for the plugin @ktoso

File: pom.xml (hunk 5 above)
Comment (chbatey) [reply pom.xml:24]: 👍

**Comment by chbatey** [comment]
Snuck in switching to a 11 JRE

**Pull request merged**